projects/igniteui-angular/src/lib/services/overlay/utilities.ts

```diff
@@ -189,6 +189,15 @@ export interface ConnectedFit {
     verticalOffset?: number;
 }
 
+// TODO: add here also EnvironmentInjector, Element (as hos element) and Nodes[][] (as projectable nodes)
+// and use this parameter when dynamic component is created.
+export interface OverlayCreateSettings extends OverlaySettings {
```

Comment:
Please include `OverlayCreateSettings` in the export from `overlay/utilities` in the services `public_api.ts`

---

Also, I think the extended object obviously means we _could_ extend it further. The TODO makes it sound like we must, yet we might never find need for the env injector for example so that comment is really not needed imo.